src/pudl/output/ferc1.py

```diff
@@ -16,74 +16,104 @@
 
 import pudl
 from pudl.transform.ferc1 import (
-    CalculationChecks,
-    CalculationGroupChecks,
-    CalculationMetricTolerance,
+    GroupMetricChecks,
+    GroupMetricTolerances,
+    MetricTolerances,
 )
 
 logger = pudl.logging_helpers.get_logger(__name__)
 
 
-EXPLOSION_CALCULATION_TOLERANCES: dict[str, CalculationChecks] = {
-    "income_statement_ferc1": CalculationChecks(
-        group_checks=CalculationGroupChecks(
-            ungrouped=CalculationMetricTolerance(
-                error_frequency=0.20, null_calculation_frequency=1.0
+EXPLOSION_CALCULATION_TOLERANCES: dict[str, GroupMetricChecks] = {
```

Comment:
Curious if you have any initial thoughts about the errors observed in the exploded tables?

Reply:
no i didn't investigate deeply. mostly was just working on the conversion itself, not the errors